### PR TITLE
Feature/qr venmo escape path

### PR DESCRIPTION
**Pull request by fedperezp**
Adding escape path for Venmo QR on Desktop
https://venmoinc.atlassian.net/browse/VMORECOMBO-173

Linked PRs
https://github.com/paypal/paypal-checkout-components/pull/1734

### CODE REVIEW

File: src/qrcode/qrcard.jsx [hunk 1]
```diff
@@ -54,6 +55,11 @@ function QRCard({
         return state === QRCODE_STATE.ERROR;
     };
 
+    const handleClick = (event : string) => {
+        window.xprops.hide();
+        window.xprops.onEscapePath(event);
```
Comment (mnicpt) [inline src/qrcode/qrcard.jsx:60]: Why are you calling this `event`?  Looks like you're passing in funding instrument.

File: src/qrcode/qrcard.jsx [hunk 2]
```diff
@@ -54,6 +55,11 @@ function QRCard({
         return state === QRCODE_STATE.ERROR;
     };
 
+    const handleClick = (event : string) => {
+        window.xprops.hide();
```
Comment (mnicpt) [inline src/qrcode/qrcard.jsx:59]: Are we wanting to `hide` or `close` here?  Do we plan on re-showing?

Comment (fedperezp) [reply src/qrcode/qrcard.jsx:59]: we don't plan on re-showing, I need to reuse the payment so I was worried that xprops.close would cancel the payment. I'm going to test with close() if you think is better in this case

Comment (fedperezp) [reply src/qrcode/qrcard.jsx:59]: Hey Steve, `close` is going the cancel the switch I'm making to the other payment flow once I go into an escape path so I think `hide` is the only alternative for now

Comment (mnicpt) [reply src/qrcode/qrcard.jsx:59]: @fedperezp So why not have the switch to the other payment return a Promise and then close?

**Pull request merged**